README.md

```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+# Azure Container Compute Upstream Projects
+
+This list of projects is maintained by the Azure container compute upstream team (TODO: how to contact the team). This list is inteded to help you make informed decisions about what projects to use (or not use) in the context of your goals (e.g. proof of concept vs. production). To make this decision you need to consider your goals, your need for formal support, the project's [maturity](#Maturity), governance, version level, and your willingness to work in open source.
+
+## Support
+
+Projects listed on this page are open source that Microsoft maintain or contribute to. These projects are [**NOT** covered by the Microsoft Azure support policy](https://support.microsoft.com/en-us/help/2941892/support-for-linux-and-open-source-technology-in-azure). To get help please search open issues using the links. If your issue isn't already represented, please open a new one. Consider that if you consume one of these projects as a part of a Microsoft or Azure product or service, you may be eligible for [support under that product](https://support.microsoft.com/en-us/hub/4343728/support-for-business).
```

Comment:
The "Consider that..." sentence (the last one) seems to contradict the one about the projects not being Azure-supported.

Reply:
I'm trying to leave room for the product or service to support. I'm open to suggestions

Reply:
how about "you may be eligible for additional business support in lieu of project-specific support".